lib/screens/app/wallet/receive.dart

```diff
@@ -286,7 +286,7 @@ class _ProductListFormState extends State<ProductListForm> {
                 product.currency == SEEDS
                     ? Container()
                     : Text(
-                        product.price.fiatFormatted! + " " + product.currency!,
+                        product.price!.fiatFormatted + " " + product.currency!,
```

Comment:
why does price need a !

Can it be null? If so what happens when it's null?

Reply:
This is v1. Its not used, i added the ! to make the compiler happy.

Reply:
Oh its in v1 code. cool.